chapter_multilayer-perceptrons/numerical-stability-and-init_vn.md

```diff
@@ -289,7 +289,13 @@ We recommend that the interested reader take a closer look at what is offered as
 Perhaps you may come across a recent clever idea and contribute its implementation to MXNet, or you may even invent your own scheme!
 -->
 
-*dịch đoạn phía trên*
+Lí do được đưa ra ở phần trên hiếm khi ảnh hưởng nhiều đến những kĩ thuật hiện đại trong việc khởi tạo tham số.
+Thực tế thì MXNet có hẳn cả một module [`mxnet.initializer`](https://mxnet.apache.org/api/python/docs/api/initializer/index.html) thực thi hàng chục các phương pháp thực nghiệm.
+Hơn thế nữa thì khởi tạo tiếp tục là một lĩnh vực nghiên cứu nóng hổi về tìm tòi trong nghiên cứu về học thuyết cơ bản trong việc tối ưu mạng neuron.
```

Comment:
```suggestion
Hơn nữa, khởi tạo liên tục (*initialization continues*) là một lĩnh vực có sự khởi sắc về tìm tòi trong nghiên cứu về học thuyết cơ bản trong việc tối ưu mạng nơ-ron.
```
Chăm xem **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** vào nha em.

Reply:
Dạ vâng ạ, e có check nhưng chắc bỏ sót cái đó ạ, em cám ơn 😍

Reply:
Ừa, em solve những suggestion của anh đã nhé. Cái nào chưa đồng thuận thì comment để trao đổi thêm, cái nào đồng thuận rồi thì sử dụng Add suggestion to batch hoặc Commit suggestion để solve.